indy-besu/network/infra/blockscout/envs/common-blockscout.env

```diff
@@ -0,0 +1,278 @@
+# DOCKER_TAG=
+ETHEREUM_JSONRPC_VARIANT=geth
+ETHEREUM_JSONRPC_HTTP_URL=http://host.docker.internal:8545/
```

Comment:
Please clean this file out of unused variables

Reply:
Done